src/bokeh/util/serialization.py

```diff
@@ -277,6 +278,22 @@ def make_globally_unique_id() -> ID:
     '''
     return ID(str(uuid.uuid4()))
 
+def make_globally_unique_css_safe_id() -> ID:
+    ''' Return a globally unique CSS-safe UUID.
+
+    Some situations, e.g. id'ing dynamically created Divs in HTML documents,
+    always require globally unique IDs. ID generated with this function can
+    be used in APIs like ``document.querySelector("#id")``.
+
+    Returns:
+        str
+
+    '''
+    while True:
+        id = make_globally_unique_id()
+        if id[0].isalpha():
+            return id
```

Comment:
I know this is not actually ever likely to an issue in practice but I'd still be more comfortable with a `while True` if it at least emitted a warning after some N iterations.